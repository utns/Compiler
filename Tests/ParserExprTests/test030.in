ar[b[1], 2]