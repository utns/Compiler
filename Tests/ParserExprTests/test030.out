____2
     ____1
____[
     ____b
[
___ar
